common/_locales/en/messages.json

```diff
@@ -1305,7 +1305,7 @@
 		"description": "This is the description of FPD shown in options. Please do not modify the template string in the message but translate the paragraphs in the placeholders section. If you find necessary, you can remove or add paragraphs.",
 		"placeholders": {
 			"paragraph1": {
-				"content": "Fingerprint Detector provides a mechanism that informs users about fingerprinting activity on visited web pages. The detector can also prevent web pages from extracting browser fingerprints if a user chooses to do so. See <a href='https://jshelter.org/fpdetection/'>our blog post</a> or <a href='https://arxiv.org/pdf/1905.01051.pdf'>Browser Fingerprinting: A survey</a> for a closer description of browser fingerprinting.",
+				"content": "Fingerprint Detector provides a mechanism that informs users about fingerprinting activity on visited web pages. The detector can also prevent web pages from extracting browser fingerprints if a user chooses to do so. See our <a href='https://jshelter.org/fpdetection/'>blog post</a> or <a href='https://arxiv.org/pdf/1905.01051.pdf'>Browser Fingerprinting: A survey</a> for a closer description of browser fingerprinting.",
```

Comment:
Is there any reasoning why you think that "our" should not be a part of the link text?

Reply:
It's different in strings [1137](https://github.com/polcak/jsrestrictor/blob/9db389c88ec4fa1a0751eec39af4be7a93652cc8/common/_locales/en/messages.json#L1137C1-L1137C1) and [1309](https://github.com/polcak/jsrestrictor/blob/9db389c88ec4fa1a0751eec39af4be7a93652cc8/common/_locales/en/messages.json#L1309C4-L1309C86), I wanted to get more uniformity.

Reply:
Thanks for pointing that out. I want to make clear what is our work and what is the work of others. So I included "our" in the link in both strings.